packages/e2e-test-app/test/__snapshots__/ViewComponentTest.test.ts.snap

```diff
@@ -1123,6 +1073,7 @@ exports[`ViewTests Views can have rounded borders 1`] = `
   "HorizontalAlignment": "Stretch",
   "Left": 0,
   "Margin": "0,0,0,0",
+  "Padding": "0,0,0,0",
```

Comment:
Grid has a `Padding` property, where as the base Panel derived ViewPanel did not, which is why we see these Padding values.